### PR TITLE
reduce scope of kubernetes manager

### PR DESCRIPTION
**Pull request by vxav**
I need the custom managers to watch `helm/cloud-provider-vsphere/Chart.yaml` but not the kubernetes manager.

So I _think_ this should do it.

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -7,10 +7,14 @@
     ":warning: Please do not merge directly :warning:",
     "Follow update procedure in the [README](https://github.com/giantswarm/cloud-provider-vsphere-app?tab=readme-ov-file#how-to-update-the-charts-automatically)."
   ],
-  "ignorePaths": [
```
Comment (AverageMarcus) [inline renovate.json5:10]: Previously this ignored ALL updates in these directories. Your change will now only ignore Kubernetes manifests. I'm not sure that's what you want is it?

File: renovate.json5 [hunk 2]
```diff
@@ -7,10 +7,14 @@
...
-    "helm",
-    "hack"
-  ],
+  "kubernetes": {
+    "ignorePaths": [
+      // We don't want to watch the K8s manifests directly.
+      "helm",
+      "config",
```
Comment (AverageMarcus) [inline renovate.json5:14]: This seems weird. Why would we NEVER want to update anything in a `config` directory?

**Review by AverageMarcus — APPROVED** [review]
🤦  I'm an idiot. Ignore my other comments, I thought this PR was in `renovate-presets`

**Comment by vxav** [comment]
I wouldn't dare making such drastic changes in the presets repo 😄

**Pull request merged**